.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -35,26 +35,27 @@ jobs:
           PR_NUMBER: ${{ github.event.number }}
         name: "Set milestone on related issues in brave-browser"
         run: |
-          URL_REGEX='(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
+          BB_ISSUE_URL_REGEX='(http(s)?:\/\/)?github.com/brave/brave-browser/issues/\d+'
```

Comment:
Will this always work?

You can do
```
Resolves url-to-issue
```
or
```
Resolves #issueNumber
```
Wouldn't this fail in the latter case?

And even if we do always get a url, shouldn't we narrow it down to url patterns matching github issues?